tests/functional/v2/config/GB.config.js

```diff
@@ -0,0 +1,39 @@
+const GB = {
+    DEV_US_SHORT_TERM: {
+        country: 'GB',
+        description: 'GB merchant eligible for short term only',
+        minAmount: 30,
+        maxAmount: 1500,
+        amounts: [
+            {
+                value: '0.00',
+                message: 'Non-qualifying Pay in 3',
+                expectedValue: 'Pay in 3 interest-free payments',
+                modalContent: {
+                    subheadline: 'Available for purchases of $30.00 to $1,500.00. No sign-up fees or late fees.',
```

Comment:
```suggestion
                    subheadline: 'Available for purchases of £30.00 to £2,000.00. No sign-up fees or late fees.',
```
An easy way to grab the correct content is by opening up the modal and copying the subheadline. That way you always know you have the correct content.